apps/studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -73,7 +73,10 @@ const ComputeInstanceSidePanel = () => {
 
   const snap = useSubscriptionPageStateSnapshot()
   const visible = snap.panelKey === 'computeInstance'
-  const onClose = () => snap.setPanelKey(undefined)
+  const onClose = () => {
+    router.push(router.asPath.split('?')[0], undefined, { shallow: true })
```

Comment:
Did you try using `router.pathname`? I think it doesn't include query params.

Reply:
```js
router.push(`/projects/${router.query.ref}/`)
```

Reply:
use `asPath`, includes the params.
https://nextjs.org/docs/pages/api-reference/functions/use-router#router-object